packages/core/helper-plugin/src/components/tests/ConfirmDialog.test.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import { lightTheme, ThemeProvider } from '@strapi/design-system';
+import { render as renderRTL, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from 'react-intl';
+
+import { ConfirmDialog, ConfirmDialogProps } from '../ConfirmDialog';
+
+describe('ConfirmDialog', () => {
+  const render = (props?: Partial<ConfirmDialogProps>) => ({
+    ...renderRTL(
+      <ConfirmDialog
+        isOpen
+        onConfirm={jest.fn()}
+        onToggleDialog={jest.fn()}
+        bodyText={{
+          id: 'app.components',
+          defaultMessage: 'Are you sure you want to unpublish it?',
+        }}
+        title={{ id: 'app.components.ConfirmDialog.title', defaultMessage: 'Confirmation' }}
+        rightButtonText={{ id: 'global.save', defaultMessage: 'Save' }}
```

Comment:
button name "Confirm" would have been more apt here, but its a test case so not a big issue.

Reply:
Those are the original props from the jsx document i just copied over. Confirm is the default text if no value is given, believe the previous tester set the message as Save instead to test if that worked.